build/constants.yaml

```diff
@@ -116,6 +116,7 @@
     node8-version: 8.17.0
     node10-version: 10.19.0
     node12-version: 12.16.1
```

Comment:
Seems like there is a new version of 12 (12.16.3). It is your call if you want to update that also as part of this PR.

Reply:
Oh I see, thanks for checking it, will update them together then. It's the latest LTS version so going to replace 12.16.1 with it.

Reply:
Oh a little correction, not replacing with the new version, will add it to the existing versions.